src/Core/Components/Wizard/FluentWizardStep.razor.cs

```diff
@@ -183,4 +184,43 @@ internal async Task InvokeOnSubmitForEditFormsAsync()
             await editForm.Key.OnSubmit.InvokeAsync(editForm.Value);
         }
     }
+
+    private async Task OnClickHandlerAsync()
+    {
+        if (!IsStepClickable)
+        {
+            return;
+        }
+
+        await FluentWizard.GoToStepAsync(Index, validateEditContexts: Index > FluentWizard.Value);
+    }
+
+    private bool IsStepClickable
```

Comment:
We don't do `Is...` or `Has...` naming, right?

Reply:
It's my private method :-) The code is clearer with this